### PR TITLE
feat: yearn staking icons

### LINKED ISSUE #814: Yearn staking during approval seeing USDC icon for an asset
Yearn approval for all assets is showing the USDC symbol instead of the asset that is selected.
USDC

Acceptance Criteria:
-Fix the issue with the approval page on the earn page is showing USDC for all assets instead of the asset that is selected
-Once fixed, have ops test your changes.
-Push to prod

### PR DESCRIPTION
**Pull request by amitojsingh366**
## Description

Fixes issue #814 where the icons for different coins wouldn't show up on the yearn staking deposit approve page

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #814

### CODE REVIEW

File: src/assets/translations/messages-en.json [hunk 1]
```diff
@@ -431,7 +431,7 @@
       "success": {
         "encryptingWallet": "Encrypting your wallet... if your browser asks to store data in persistent storage, please click 'Allow'.",
         "header": "Wallet Connected",
-        "success": "Your wallet has been connected",
+        "success": "You can now close this window or it will auto-close shortly",
```
Comment (0xdef1cafe) [inline src/assets/translations/messages-en.json:434]: you're definitely not up to date with develop

Comment (amitojsingh366) [reply src/assets/translations/messages-en.json:434]: how is this even possible, I was the one that introduced that change in the translations even 🤯

Comment (amitojsingh366) [reply src/assets/translations/messages-en.json:434]: wait a minute, something is wrong here

Comment (amitojsingh366) [reply src/assets/translations/messages-en.json]: fixed now lol

File: src/components/AssetIcon.tsx [hunk 2]
```diff
@@ -10,10 +10,11 @@ type AssetIconProps = {
 
 // The icon here is as the placeholder while the icon loads, or if it fails to load.
 
-export const AssetIcon = ({ symbol, ...rest }: AssetIconProps) => {
+export const AssetIcon = ({ symbol, src, ...rest }: AssetIconProps) => {
```
Comment (0xApotheosis) [inline src/components/AssetIcon.tsx:13]: We've added `src?: string`, but not updated the types in `AssetIconProps` accordingly.

We might also want to add a comment noting the behavior of `AssetIcon`, i.e. that either `symbol` or `src` should be supplied as arguments, but not both. If both are supplied, we use `src`.

**Review by 0xApotheosis — CHANGES_REQUESTED** [review]
A few comments, looking good.

**Comment by 0xApotheosis** [comment]
@amitojsingh366 could you please also ensure that all PRs have the description section filled out?

This helps your reviews know what change you are expecting to have produced in the PR, as well as with ensuring useful commit history when the PR is squashed and merged.

File: src/components/AssetIcon.tsx (hunk 2 above)
Comment (amitojsingh366) [reply src/components/AssetIcon.tsx:13]: @0xApotheosis actually `src?: string` is inherited from `AvatarProps`

File: src/components/AssetIcon.tsx [hunk 3]
```diff
@@ -10,10 +10,11 @@ type AssetIconProps = {
 
 // The icon here is as the placeholder while the icon loads, or if it fails to load.
```
[the remark this answers is not in the document]
Comment (amitojsingh366) [reply src/components/AssetIcon.tsx]: done

**Comment by amitojsingh366** [comment]
> @amitojsingh366 could you please also ensure that all PRs have the description section filled out?
> 
> This helps your reviews know what change you are expecting to have produced in the PR, as well as with ensuring useful commit history when the PR is squashed and merged.

done

**Comment by amitojsingh366** [comment]
> @amitojsingh366 could you please also ensure that all PRs have the description section filled out?
> 
> This helps your reviews know what change you are expecting to have produced in the PR, as well as with ensuring useful commit history when the PR is squashed and merged.

@0xApotheosis done

File: src/components/AssetIcon.tsx (hunk 2 above)
Comment (0xApotheosis) [reply src/components/AssetIcon.tsx:14]: Perfect, leave as is 👌

**Review by 0xApotheosis — APPROVED** [review]
LGTM, tested and works as expected:

Screen Shot 2022-02-02 at 6 58 09 am

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**